elide-core/src/main/java/com/yahoo/elide/jsonapi/links/DefaultJsonApiLinks.java

```diff
@@ -52,16 +52,10 @@ public Map<String, String> getRelationshipLinks(PersistentResource resource, Str
     protected String getResourceUrl(PersistentResource resource) {
         StringBuilder result = new StringBuilder();
 
-        if (StringUtils.isEmpty(baseUrl)) {
-            if (resource.getRequestScope().getBaseUrlEndPoint() != null) {
-                result.append(resource.getRequestScope().getBaseUrlEndPoint());
-                String jsonApiPath = resource.getRequestScope().getElideSettings().getJsonApiPath();
-                if (StringUtils.isNotEmpty(jsonApiPath)) {
-                    result.append(jsonApiPath);
-                }
-                result.append("/");
-            }
-        } else {
+        if (resource.getRequestScope().getBaseUrlEndPoint() != null) {
+            result.append(resource.getRequestScope().getBaseUrlEndPoint());
+            result.append("/");
```

Comment:
Did we get rid of the json api path here because base URL now includes it?

Reply:
Yes the base url now includes it.

Reply:
Looked briefly at the code.  It looks to me like the request scope base URl is populated by the controller - but the controller just pulls the baseUrl from the settings object (and does not include the JSON API or GraphQL path here).  What am I missing?